pkg/samplerepo/assets/sample/README.md.tmpl

```diff
@@ -29,7 +29,7 @@ If you're running lakeFS with Docker then all the tools you need (`lakectl`, `du
 ```bash
 docker run --name lakefs \
            --rm --publish 8000:8000 \
-           treeverse/lakefs:0.100.0-duckdb \
+           treeverse/lakefs:0.101.0-duckdb \
```

Comment:
Maybe the version can also be templeted somehow ?

Reply:
it's moot since we're moving to `latest-duckdb` tag imminently, this PR is just to tide us over